packages/amplify-category-function/package.json

```diff
@@ -22,6 +22,9 @@
     "amplify",
     "aws"
   ],
+  "publishConfig": {
```

Comment:
Do we want to make this package public?

Reply:
It is already public: https://www.npmjs.com/package/amplify-category-function